Makefile

```diff
@@ -43,8 +43,8 @@ collectorpb/collector.pb.go:
 else
 collectorpb/collector.pb.go: lightstep-tracer-common/collector.proto
 	docker run --rm -v $(shell pwd)/lightstep-tracer-common:/input:ro -v $(shell pwd)/collectorpb:/output \
-	  lightstep/protoc:latest \
-	  protoc --go_out=plugins=grpc:/output --proto_path=/input /input/collector.proto
+	  lightstep/grpc-gateway:latest \
```

Comment:
@djspoons can you help me release my new docker image so that this can be build externally?